README.md

```diff
@@ -285,7 +285,7 @@ KAFKA|2.7.1, 2.8.1, 3.0.0|
 KAFKA_REST_PROXY|5.0.0|KAFKA
 KAFKA_SCHEMA_REGISTRY|5.0.0, 5.0.4|KAFKA
 OPENSEARCH|1.0.0
-ELASTICSEARCH|1.8.0, 1.11.0
+ELASTICSEARCH|1.13.3
```

Comment:
**[Question]** Is reason we don't support the prior versions because of vulnerabilities mentioned in the original ticket INS-16948?